src/Power_Up_Sequence.md

```diff
@@ -1,62 +1,401 @@
+1. Find the title checksum [in a table](https://github.com/ISSOtm/gb-bootroms/blob/443d7f057ae06e8d1d76fa8083650cf0be2cd0ae/src/cgb.asm#L1221-L1230), and record its index within the table.
+
+   An almost-complete list of titles corresponding to the different checksums can be found in [Liji's free CGB boot ROM reimplementation](https://github.com/LIJI32/SameBoy/blob/1d7692cff5552e296be5e1ab075c4f187f57132c/BootROMs/cgb_boot.asm#L230-L328).
+   * If not found, palettes ID $00 is used.
+   * If the index is 64 or below, the index is used as-is as the palettes ID, and the algorithm ends.
+   * Otherwise, it must be further corrected based on the title's fourth letter; proceed to the step below.
+1. The fourth letter is searched for in [another table](https://github.com/ISSOtm/gb-bootroms/blob/443d7f057ae06e8d1d76fa8083650cf0be2cd0ae/src/cgb.asm#L1232-L1240).
+   * If the letter can't be found, palettes ID $00 is used.
+   * If the letter is found, the index obtained in the previous step is increased by 14 times the row index to get the palettes ID.
+     (So, if the letter was found in the first row, the index is unchanged; if it's found in the second row, it's increased by 14, and so on.)
+
+The resulting palettes ID is used to pick 3 palettes out of a table via a fairly complex mechanism.
+The user can override this choice using certain button combinations during the logo animation; some of these manual choices are identical to auto-colorizations, [but others are unique](https://tcrf.net/Notes:Game_Boy_Color_Bootstrap_ROM#Manual_Select_Palette_Configurations).
+
+::: tip
+
+A table of checksums (and tie-breaker fourth letters when applicable) and the corresponding palettes can be found [on TCRF](https://tcrf.net/Notes:Game_Boy_Color_Bootstrap_ROM#Assigned_Palette_Configurations).
+
+:::
+
+If the ID is either $43 or $58, then the Nintendo logo's tilemap is written to VRAM.
+This is intended for games that perform some kind of animation with the Nintendo logo; it suddenly appears in the middle of the screen, though, so it may look better for homebrew not to use this mechanism.
+
+### Scrapped palette switching menu
+
+Remnants of a functionality designed to allow switching the CGB palettes while the game is running exist in the CGB CPU.
+
+## Stadium 2
+
+Pokémon Stadium 2's "GB Tower" emulator contains a very peculiar boot ROM.
+It can be found at offset $015995F0 in the US release, and is $3F0 bytes long.
+Its purpose is unknown.
+
+This boot ROM does roughly the same setup as a regular CGB boot ROM, but writes to $FF50 very early, and said write is followed by a lock-up loop.
+Further, the boot ROM contains a valid header, which is mostly blank save for the logo, compatibility flag (which indicates dual compatibility), and header checksum.
+
+## Logo check
+
+While it may make sense for the boot ROM to at least partially verify the ROM's integrity via the header check, why does it check the logo's integrity separately?
```

Comment:
I don't see why this needs to be worded as a direct question to the reader, also supposing known information (it may make sense to at least partially verify? why?)